thirdparty/fast_noise_2/src/FastSIMD/FastSIMD.cpp

```diff
@@ -215,7 +215,7 @@ CLASS_T* SIMDLevelSelector( FastSIMD::eLevel maxSIMDLevel, FastSIMD::MemoryAlloc
             return nullptr;
         }
 
-        return SIMDLevelSelector<CLASS_T, FastSIMD::SIMDTypeList::GetNextCompiledAfter<SIMD_LEVEL>>( maxSIMDLevel, allocator );        
```

Comment:
Don't modify FastNoise2. It's a thirdparty library. Warnings are turned off on it for that reason. If you still can't compile it, you should change compilation flags specifically for its files, or refer to the library's author maybe?

Reply:
Actually, looks like theyr'e *not* turned off. At least, not for MSVC; only a single warning is disabled for that thirdparty library. I'll just make it disable warnings universally instead